packages/website/src/features/Deploy/QueueFromGitOpsPage.tsx

```diff
@@ -1027,7 +1027,8 @@ export default function QueueFromGitOps() {
             <Box>
               <NoncePicker safe={currentSafe} handleChange={setPickedNonce} />
               <HStack gap="6">
-                {stager.execConditionFailed ? (
+                {stager.execConditionFailed &&
+                (buildState.result?.safeSteps.length || 0) > 0 ? (
```

Comment:
Perfect, this check should also be added on the "Execute Outside Safe Txns", because if not it would allow the user to deploy those contracts but not add them to any package.